driver-core/src/main/java/com/datastax/driver/core/AbstractData.java

```diff
@@ -139,13 +139,21 @@ public T setLong(String name, long v) {
     }
 
     public T setDate(int i, Date v) {
```

Comment:
Since getDate is being replaced with getTimestamp, should't we just get rid of setDate since there is no symmetric get for it?